.github/workflows/release.yml

```diff
@@ -36,25 +34,37 @@ jobs:
           node-version: 18
 
       - run: npm install
-
       - run: npm run build
-
       - run: npm run test
         if: ${{ inputs.skip-test == false }}
+      - name: Upload release artifacts
+        id: upload-release
+        uses: actions/upload-artifact@v4
+        with:
+          name: release
+          path: lib
```

Comment:
We don't make this configurable as all packages have been changed to build to the lib folder. If we made use of the publish-packages input then it would require a PR in a lot of other packages.